src/helpers/undo_helpers.sh

```diff
@@ -2,10 +2,29 @@
 
 
 function add_undo_steps {
-  prepend_to_file "$1" "$UNDO_STEPS_FILE"
+  local steps=$1
+
+  prepend_to_file "$steps" "$UNDO_STEPS_FILE"
+}
+
+
+# Returns the post_undo steps for the given command
+function post_undo_steps_for {
+  local command=$1
+
+  local step_with_arguments
+  read -a step_with_arguments <<< "$command" # Split string into array
+
+  local step="${step_with_arguments[0]}"
+  local arguments="${step_with_arguments[*]:1}"
+  local fn="post_undo_steps_for_$step"
+  if [ "$(type "$fn" 2>&1 | grep -c 'not found')" = 0 ]; then
+    eval "$fn $arguments"
+  fi
```

Comment:
This has is almost a full duplication of `undo_steps_for` aside from the definition of `fn`